src/css/styles.css

```diff
@@ -25,12 +26,32 @@ body {
     border-style: dashed;
 }
 
+#mainContainer {
+    width: 100%;
+    position: absolute;
+    bottom: 0;
+}
+
 .itemArea {
     background: #fff;
     overflow-y: auto;
     padding-top: 10px;
 }
 
+#itemsContainer {
+    display: flex;
+}
+
+#itemsArea {
```

Comment:
Is there a compelling reason for hard-coding layout sizes instead of using the responsive elements in Bootstrap?

Reply:
Yes, the primary reason is because Bootstrap's grid system presents 2 challenges for our UI:
1. At certain page widths (the most common from what I've seen), Bootstrap's grid system places a margin on the left/right side of the main content area (centered) and leaves a fair amount of space left unavailable.  This is particularly relevant with regards to JSON viewing/editing.  The new code removes this unused space.
2. The Bootstrap grid framework defaults to a float implementation when screen size is reduced (i.e. to a typical mobile size) which either causes the right `#dropArea` to float down (below the `#itemArea` , and thus become unreachable) or makes either column unusable because the column elements are too small.  It also causes the `navbar` to grow in height and thus cover up the top portion of either content columns.

Basically, there are lots of issues with our UI/UX css currently, and these changes are meant to address some of them before we get to the point where we can look at a full UI refresh down the road.

Reply:
The layout changes you're proposing will have implications across the entire app. They should be broken out to a separate branch and PR so we can have a more focused discussion on those specific changes.

Regarding your comments, screenshots would really be helpful. Both 1 and 2 should be addressable by making better use Bootstrap's [media breakpoints](https://getbootstrap.com/docs/3.3/css/#grid-options). I'm not opposed to migrating away from those in favor of something else as long as it provides flexibility and responsiveness. Hardcoded css rules make both of those harder.